### PR TITLE
No longer count skipped/disabled tests as failures.

### PR DESCRIPTION
**Pull request by lukewrites**
Disabled tests are counted as passes, skipped tests are counted as passes, failed tests are counted as failures. Also updates how test results are output to the console:

image
image

Snag_6335a9c
Snag_63370f2
Snag_6338092

csv:
image

### CODE REVIEW

**Comment by lriggs** [comment]
Does this fix the csv and json files as well?

**Comment by lukewrites** [comment]
@lriggs updates the csv but not the JSON; I'll update it to do that.

File: tdvt/tdvt/tdvt.py [hunk 1]
```diff
@@ -241,6 +247,8 @@ def enqueue_failed_tests(run_file: Path, root_directory, args, rt: RunTimeTestCo
     all_tdvt_test_configs = {}
     all_test_pairs = []
     failed_tests = tests['failed_tests']
+    skipped_tests = tests['skipped_tests']
+    failed_and_skipped_tests = {**failed_tests, **skipped_tests}
```
Comment (lukewrites) [inline tdvt/tdvt/tdvt.py]: need to remove this, it's not used.

File: tdvt/tdvt/tdvt.py [hunk 2]
```diff
@@ -531,7 +543,7 @@ def run_tests_impl(tests: List[TestSet], max_threads, args):
 
     for test_set, test_config in tests:
         runner = TestRunner(test_set, test_config, lock, args.verbose, len(all_work) + 1)
-        if test_set.smoke_test and not test_set.test_is_enabled is False:
```
Comment (lukewrites) [inline tdvt/tdvt/tdvt.py]: removed so disabled tests will show up in json

File: tdvt/tdvt/tdvt.py [hunk 3]
```diff
@@ -555,25 +567,31 @@ def run_tests_impl(tests: List[TestSet], max_threads, args):
...
             failing_ds = set(item.test_set.ds_name for item in smoke_tests if item.failed_tests > 0)
             print("{} smoke test(s) failed. Please check logs for information.".format(failed_smoke_tests))
             if require_smoke_test:
-                print("Smoke tests failed, exiting.")
+                print("\nSmoke tests failed, exiting.")
                 sys.exit(1)
 
         if require_smoke_test:
-            print("Smoke tests failed, exiting.")
+            print("\nSmoke tests finished. Exiting.")
```
Comment (lukewrites) [inline tdvt/tdvt/tdvt.py]: think this was a bug before.

**Comment by lukewrites** [comment]
`csv` and `json` output both look right, test counts look right too.

I've updated the command line output a little bit, now it looks like this:
image
thoughts?

**Comment by lukewrites** [comment]
image
is nicer, but needs rewording

**Pull request merged**